content/en/docs/concepts/workloads/pods/ephemeral-containers.md

```diff
@@ -76,3 +76,195 @@ you can view processes in other containers.
+
+## Pod Security Policies
+
+Pod Security Policies used to be the default policy tool for Kubernetes, and a lot of projects have rules based on Pod Security Policies (PSP). However if you are relying on PSP in a modern cluster you should really start looking for other options like OPA Gatekeeper or Kyverno. PSP has been deprecated since Kubernetes v1.21 and will be removed in v1.25.
+
+If PSP is your only policy tool and you are planning to upgrade to v1.23, don't. As PSP is deprecated no new features have been added, and that includes policy enforcement on ephemeral containers. Which means that any security context in an ephemeral container is allowed no matter the PSP in the cluster. The PSP below will have no affect when adding an ephemeral container to a Pod which is privileged.  
+   
+apiVersion: policy/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  name: default
+  spec:
+    privileged: false
+      seLinux:
+          rule: RunAsAny
+            supplementalGroups:
+                rule: RunAsAny
+                  runAsUser:
+                      rule: RunAsAny
+                        fsGroup:
+                            rule: RunAsAny
+                              volumes:
+                                - '*'
+
+
+
+## RBAC
+
+Disallowing ephemeral containers with RBAC could be an option if the feature is not needed and it is not possible to disable the feature completely. The KEP-277: Ephemeral Containers state the following about using RBAC to disable ephemeral containers.
+
+Cluster administrators will be expected to choose from one of the following mechanisms for restricting usage of ephemeral containers:
+
+Use RBAC to control which users are allowed to access the /ephemeralcontainers subresource.
+Write or use a third-party admission controller to allow or reject Pod updates that modify ephemeral containers based on the content of the update.
+Disable the feature using the EphemeralContainers feature gate.
+RBAC is additive which means that it is not possible to remove permissions from a role. This type of mitigation obviously does not matter if all users a cluster admin, which they should not be, so we assume that new roles are created for the cluster consumers. In this case having a look at the existing roles can be enough to make sure that the subresource /ephemeralcontainers is not included in the role.
+
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: edit
+  rules:
+  - apiGroups:
+    - ""
+      resources:
+        - pods
+          - pods/attach
+            - pods/exec
+              - pods/portforward
+                - pods/proxy
+                  verbs:
+                    - create
+                      - delete
+                        - deletecollection
+                          - patch
+                            - update
```

Comment:
```suggestion
```yaml
apiVersion: rbac.authorization.k8s.io/v1
kind: ClusterRole
metadata:
  name: edit
  rules:
  - apiGroups:
    - ""
      resources:
        - pods
          - pods/attach
            - pods/exec
              - pods/portforward
                - pods/proxy
                  verbs:
                    - create
                    - delete
                    - deletecollection
                    - patch
                    - update
```
```
Not sure the indentations correct or not. Seems weird.